### PR TITLE
[Draft][Do not merge] Add release notes

### PR DESCRIPTION
**Pull request by kfaraz**

### CODE REVIEW

File: docs/release_notes_25.md [hunk 1]
```diff
@@ -0,0 +1,664 @@
...
+When MSQ submits tasks containing SQL with sensitive keys, the keys can get logged in the file.
+Druid now masks the sensitive keys in the log files using regular expressions.
+
+https://github.com/apache/druid/pull/13231
+
+### <a name="25.0.0-multi-stage-query-task-engine-query-history" href="#25.0.0-multi-stage-query-task-engine-query-history">#</a> Query history
+
+Multi-stage queries no longer show up in the Query history dialog. They are still available in the **Recent query tasks** panel.
+
+## <a name="25.0.0-query-engine" href="#25.0.0-query-engine">#</a> Query engine
```
Comment (cryptoe) [inline docs/release_notes_25.md:142]: I think this heading seems wrong. We can remove this.

**Review by cryptoe — COMMENTED** [review]
One comment. Rest all looks fine to be from the MSQ side.

File: docs/release_notes_25.md (hunk 1 above)
Comment (cryptoe) [inline docs/release_notes_25.md:142]:
```suggestion
```

**Comment by adarshsanjeev** [comment]
Looks good from my side

**Comment by rohangarg** [comment]
Looks good from my side

**Comment by LakshSingla** [comment]
Looks good from my side as well. Thanks!

File: docs/release_notes_25.md [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+
+For more information, see [Front coding](https://druid.apache.org/docs/latest/ingestion/ingestion-spec.html#front-coding).
+
+https://github.com/apache/druid/pull/12277 
+
+### <a name="25.0.0-highlights-kubernetes-native-tasks" href="#25.0.0-highlights-kubernetes-native-tasks">#</a> Kubernetes native tasks
+
+Druid can now use Kubernetes to launch and manage tasks, eliminating the need for MiddleManagers.
+
+To use this feature, enable the [`druid-kubernetes-overlord-extensions`]((../extensions.md#loading-extensions) in the extensions load list for your Overlord process.
```
Comment (kfaraz) [inline docs/release_notes_25.md:34]:
```suggestion
To use this feature, enable the [druid-kubernetes-overlord-extensions]((../extensions.md#loading-extensions) in the extensions load list for your Overlord process.
```

**Pull request closed without merging**